client/src/GlobalStyle.js

```diff
@@ -41,6 +41,7 @@ h1 {
 
 canvas {
   height: 100vh;
```

Comment:
i would suggest to use a min-height, because otherwise you will have problems with resizing the window or if your content is bigger than your screen:
```suggestion
  height: 100%;
  min-height: 100vh;
```

Reply:
Thanks, I changed it. 😊